docs/Toy/toy/dialects/toy.py

```diff
@@ -232,7 +232,7 @@ def verify_(self):
         if len(self.body.blocks) != 1:
             raise VerifyException("Expected FuncOp to contain one block")
 
-        block = self.body.blocks[0]
+        block = self.body.block
```

Comment:
Technically this is semantically different, as it will now fail when there are more than one block, while it previously ignored subsequent blocks.

A more correct replacement would be `.first_block`.

Reply:
Oh I thought I had gotten rid of all of these, I'll fix this.